webapp/src/blocks.tsx

```diff
@@ -481,6 +481,8 @@ export class Editor extends toolboxeditor.ToolboxEditor {
     private prepareBlockly(forceHasCategories?: boolean) {
         pxt.perf.measureStart("prepareBlockly")
         let blocklyDiv = document.getElementById('blocksEditor');
+        if (!blocklyDiv)
```

Comment:
This was adding an extra crash because blocksEditor doesn't exist if we crash, and this happens in a separate promise thread. Can do something different if we want here?